idiglove/aintboard
# Forgot Password ability

Additional env variable:
BREVO_FORGOT_PASSWORD_TEMPLATE_ID="11"

Testing:
1. Go to /forgot-password
2. Enter email
3. Check email for reset password link
4. Input passwords
5. Click submit
6. try signing in in /signin

Edge cases:
1. Expired token (received 10 minutes earlier)
2. using an unregistered email in /forgot-password
